content/en/community/hackathons/sessions/contributing-to-unikraft/content/Overview_Warm-Up.md

```diff
@@ -1,11 +1,15 @@
 Let's check the integrity of this library using its test suite through the exposed wrapper function.
 
-For this task, you have to move, [or clone](https://github.com/unikraft/lib-libhogweed), the library in the `$UK_LIBS` folder and the `work/01-tut-porting/apps/app-libhogweed` application in the `$UK_APPS` folder.
-Fill the `TODO` lines from the application code: add the `libhogweed` library as a dependency in its `Makefile` and call from `main.c` the function exposed by the library for running the test suite.
+For this task, you have to move the library in the `$UK_LIBS` folder and the `work/01-tut-porting/apps/app-libhogweed` application in the `$UK_APPS` folder.
+You can also keep the current directory structure but set `$UK_LIBS` and `$UK_APPS` to `work/01-tut-porting/libs` and `work/01-tut-porting/apps`.
```

Comment:
Wouldn't it be better if we suggested that if the users would like to keep the current working directory, they should only clone the unikraft core repo, without having to modify the app Makefile? 

Also, I suggest that we change the name of the `libhogweed` library to `lib-libhogweed` in order to maintain consistency with `lib-newlib`, which should also be cloned as part of this exercise.

Reply:
Hi @mariasfiraiala, thanks for pointing that out.
I would actually go the other way around to remove the inconsistency, renaming `lib-newlib` to `newlib` during the cloning process.
I think most of the students will be used to this naming convention by session 05 (`lib-name` on github being just `name/` locally and used as such in the build process), since it is used since the first session ([here](https://unikraft.org/community/hackathons/sessions/baby-steps/#03-tutorial-mount-9pfs) for example).
What do you think?

Reply:
@StefanJum I don't really have a  tough stance regarding the naming convention, it's in my habits to not rename the files when cloning them.

Anyway, if we decide to go with names without `app-` or `lib-` as a prefix, I'll also need to update the session I am working on accordingly.

Reply:
About my first suggestion, what are your thoughts?

Reply:
@mariasfiraiala yes, the first suggestion is very on point, I will make an update for that.

About the naming conventions, I noticed that many parts of sessions down the road mix the names, only the first ones seem to be consistent, I guess after a while students realize the naming differences and just do their own.
I will make sure there is consistent naming throughout the entire session, no matter which convention it is.